external/gc-timings/gc_timings_stubs.c

```diff
@@ -0,0 +1,98 @@
+#include <caml/memory.h>
```

Comment:
What's the provenance of the gc_timings library?

Reply:
It was written for this PR. I can move it elsewhere if needed.